apps/guardian-ui/src/components/CustomStepper.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { Box, Center, Flex, Text } from '@chakra-ui/react';
+import { ReactComponent as CheckIcon } from '../assets/svgs/white-check.svg';
+import { StepState } from '../types';
+
+interface StepProps {
+  text: string;
+  state: StepState;
+  margin?: string;
+}
+
+const Step: React.FC<StepProps> = ({ text, state, margin = '-48px' }) => {
+  const active = state === StepState.Active;
```

Comment:
nit: I'd simplify and just use `StepState` directly instead of creating consts here